docs/docs-content/clusters/data-center/nutanix/add-nutanix-cloud-account.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_label: "Add Nutanix Cloud Account"
+title: "Add Nutanix Cloud Account"
+description: "Learn how to add a Nutanix cloud account in Palette."
+hide_table_of_contents: false
+tags: ["data center", "nutanix"]
+---
+
+ Once the Private Cloud Gateway (PCG) is installed, the Nutanix cloud account must be added to Palette. The required fields to add the cloud are pre-defined by the system administrator during the cloud registration process. These cloud-specific fields vary based on the cloud type.
+
+## Prerequisites
+
+- A Nutanix cloud that a system administrator has registered with Palette. 
+
+- A PCG installed that connects the Nutanix cloud with Palette. For guidance, refer to [Install Private Cloud Gateway](install-pcg.md).
+
+
+## Add Cloud Account
+
+1. Log in to [Palette](https://console.spectrocloud.com/) as a tenant administrator.
+
+2. From the **left Main Menu**, select **Tenant Settings**. 
+
+3. Next, select **Cloud Accounts** in the **Tenant Settings** menu.
+
+4. Locate Nutanix and click **Add Nutanix Cloud**.  
+
+5. Fill out the following input values and click on **Confirm** to continue. 
+
+  | **Field** | **Description** |
+  |-----------|-----------------|
+  | **Name**| A custom name for the account. |
+  | **Private Cloud Gateway**| This name is auto-filled based on the name you provided when you installed the PCG.|
```

Comment:
Select PCG from the list of deployed PCG in your setup

Reply:
Revised with suggested wording.